events/workshops-gbi-february-2023.html

```diff
@@ -0,0 +1,45 @@
+---
+layout: news-events
+filename: events
+title: February 2023 OME users seminars
+main-blurb: 2 online seminars (30 minutes each) given by the OME Team on OMERO and IDR organized by Global BioImaging.
+---
+     <hr class="whitespace">
+        <div class="row">
+            <div class="column">
+                <h2 class="event-day">Organisers: <a href="https://globalbioimaging.org/" target="_blank">Global BioImaging</a></h2>
+            </div>
+        </div>
+        <div class="row">
+            <div class="column">
+                <h2 class="event-day">Location: held remotely
+            </div>
+        </div>
+        <div class="row">
+            <div class="column">
+                <h2 class="event-day">Seminar 1.: <a href="https://globalbioimaging.org/international-training-courses/facility-management-2023-mexico" target="_blank">Principles of FAIR data</a>
+            </div>
+        </div>
+        <div class="row">
+            <div class="column">
+                <h2 class="event-day">Date: 24 February 2023</h2>
+            </div>
+        </div>
+        <div class="row">
+            <div class="column">
+                <h2 class="event-day">Seminar 2.: <a href="https://globalbioimaging.org/international-training-courses/facility-management-2023-mexico" target="_blank">Reuse of image data in IDR repository: Case studies</a>
+            </div>
+        </div>
+        <div class="row">
+            <div class="column">
+                <h2 class="event-day">Date: 24 February 2023</h2>
+            </div>
+        </div>
+        <hr class="whitespace">
+        <div class="row">
+            <div class="column">
+                <h2 class="event-day">Duration: 30 minutes for each of the 2 Seminars </h2>
+                <p class="event-day-description">The first seminar will introduce the FAIR principle as applied in case of Image Data Resource (IDR) and OME-NGFF. The second seminar will then go over case studies and example of data reanalysis from IDR. For more details and to apply for the workshop(s), please visit <a href="https://globalbioimaging.org/international-training-courses/facility-management-2023-mexico" target="_blank">the site of the event.
```

Comment:
```suggestion
                <p class="event-day-description">The first seminar will introduce the FAIR principles as applied in case of Image Data Resource (IDR) and OME-NGFF. The second seminar will then go over case studies and example of data reanalysis from IDR. For more details and to apply for the workshop(s), please visit <a href="https://globalbioimaging.org/international-training-courses/facility-management-2023-mexico" target="_blank">the site of the event.
```